common/static/common/js/karma.common.conf.js

```diff
@@ -167,60 +170,54 @@ function junitSettings(config) {
  * @param {String} pattern
  * @return {String}
  */
-var defaultNormalizeFunc = function(appRoot, pattern) {
+// I'd like to change fix the no-shadow violation on the next line, but it would break this shared conf's API.
+function defaultNormalizeFunc(appRoot, pattern) {  // eslint-disable-line no-shadow
     if (pattern.match(/^common\/js/)) {
         pattern = path.join(appRoot, '/common/static/' + pattern);
     } else if (pattern.match(/^xmodule_js\/common_static/)) {
         pattern = path.join(appRoot, '/common/static/' +
             pattern.replace(/^xmodule_js\/common_static\//, ''));
     }
     return pattern;
-};
+}
 
-var normalizePathsForCoverage = function(files, normalizeFunc) {
+function normalizePathsForCoverage(files, normalizeFunc, preprocessors) {
```

Comment:
I changed the function expressions to function statements in this file because it was in violation of the one-var and vars-on-top eslint rules.